python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py

```diff
@@ -333,20 +339,38 @@ def _remove_ctrl_vars(self, graph):
     def _optimize_fp32_graph(self, graph):
         graph = self._update_activations(graph)
         graph = self._remove_ctrl_vars(graph)
+        graph = self._apply_pass(graph, 'attention_lstm_fuse_pass')
+        graph = self._apply_pass(graph, 'seqconv_eltadd_relu_fuse_pass')
+        #  graph = self._apply_pass(graph, 'seqpool_concat_fuse_pass')
+        graph = self._apply_pass(graph, 'seqpool_cvm_concat_fuse_pass')
```

Comment:
Are these fueses tested to be faster than original unfused graph? If lstm did not use mkldnn, will the fuse also be fast?

Reply:
The list of passes was supplemented to reflect the the list of optimizations applied by PaddlePredictor for FP32 inference.